test/alice_test.exs

```diff
@@ -7,10 +7,6 @@ defmodule AliceTest do
 
   alias Alice.Handlers.TestHandler
 
-  test "contains a default set of handlers" do
-    assert [Alice.Earmuffs, Alice.Handlers.Help, Alice.Handlers.Utils] == Alice.handlers(%{})
-  end
-
```

Comment:
this should still pass. I think it's important that an instance of alice will always contain those built-in handlers

Reply:
The current implementation has a race condition that makes this test and the one after fail to work together sometimes due to the nature of it resetting an environment variable. Now that I think on it though, I think there's a better implementation. I'll address this.

Reply:
Added it back, there was a race condition I couldn't solve in them yesterday, and I figured the behavior was covered by the other test. But you're right it's better to be explicit. I solved the race condition and cleaned up some stuff that wasn't used while I was there in the test.